validation/shallow_water_model/shallow_water_jet_topography.jl

```diff
@@ -0,0 +1,164 @@
+using Oceananigans
+using Oceananigans.Models: ShallowWaterModel
+   
+Nx, Ny = 50, 128
+Lx, Ly = 2π,   10
+H₀     = 10
+α, β   = 0.1, -0.1
+Lb, Hb = α * Ly, β * H₀
+
+f, g, U  = 1, 1, 1
+Δη = -f * U / g 
+
+grid = RectilinearGrid(size = ( Nx, Ny),
+                          x = (  0, Lx),
+                          y = (-Ly, Ly),
+                   topology = (Periodic, Bounded, Flat))
+
+η(x, y) =   Δη * tanh(y)
+b(x, y) = - H₀ + Hb * tanh(y/Lb)  # FJP: sign in front of Hb?
+
+h̄(x, y) = η(x, y) - b(x, y)
+ū(x, y) =  U * sech(y)^2
+ 
+ uⁱ(x, y) = ū(x, y) + 1e-4 * exp(-y^2) * randn()
+uhⁱ(x, y) = uⁱ(x, y) * h̄(x, y)
+
+ū̄h(x, y) = ū(x, y) * h̄(x, y)
+
+bathymetry(x,y) = b(x,y)
+
+model = ShallowWaterModel(; grid, coriolis = FPlane(f=f), 
+                gravitational_acceleration = g,
+                               timestepper = :RungeKutta3,
+                        momentum_advection = WENO(),
+                                bathymetry = bathymetry
+                        )
+
+set!(model, uh = ū̄h, h = h̄)
+
+uh, vh, h = model.solution
+
+u = uh / h
+v = vh / h
+
+# Plot the Geostrophic State 
+
+using CairoMakie
+
+x = grid.xᶜᵃᵃ[1:Nx]
+y = grid.yᵃᶜᵃ[1:Ny]
```

Comment:
lets use nodes here?

```suggestion
x = xnodes(grid, Center())
y = ynodes(grid, Center())
```